Before: 037fdeb0f338acd3b301085ccfe5059d345794c2c8955edf3024af42ba0c2535
After: 2f51a09e8ec364cec8a95acb206c9315cefb0830081c7eb6eb073472e379dc67
Patch: d1e730400fcba615f64c5f893420bd73df865c6d85e2d3c705ee09b2efa23632
--- Gemfile.corvid	2012-07-24 17:31:31.457816507 +1000
+++ Gemfile.corvid	2012-07-24 17:33:56.276859042 +1000
@@ -1,7 +1,11 @@
 # vim:ft=ruby et ts=2 sw=2:
+header "# This file is generated by corvid update:deps"
 
-gem 'corvid'
-
+if options[:use_corvid_gem]
+  gem 'corvid', ">= #{::Corvid::VERSION}"
+else
+  gem 'corvid', path: CORVID_ROOT
+end
 gem 'rake', group: :rake
 
 group :test do
--- /dev/null	2012-07-24 17:25:00.561565760 +1000
+++ Gemfile.raven	2012-07-24 17:33:56.276859042 +1000
@@ -0,0 +1,38 @@
+# vim: ft=ruby et ts=2 sw=2 :
+header "# This file is generated by raven update:deps"
+footer "# v\im: ft=ruby et ts=2 sw=2 :"
+
+if options[:use_raven_gem]
+  gem 'raven', ">= #{::Raven::VERSION}"
+else
+  gem 'raven', path: RAVEN_ROOT
+end
+gem 'rake', group: :rake
+
+group :test do
+  gem 'simplecov'
+end if Dir.exists?('test')
+
+group :test_unit do
+  gem 'minitest'
+  gem 'guard-minitest', '>= 0.5.0'
+end if Dir.exists?('test/unit')
+
+group :test_spec do
+  gem 'rspec'
+  gem 'guard-rspec'
+end if Dir.exists?('test/spec')
+
+group :ci do
+  gem 'ci_reporter', require: false
+end
+
+group :doc do
+  gem 'yard'
+  gem 'rdiscount' # TODO aaaaaaaaaahhhhhhhhhh
+end
+
+group :debug do
+  gem 'debugger', platforms: :mri
+end
+
--- /dev/null	2012-07-24 17:25:00.561565760 +1000
+++ test/bootstrap/all.rb.tt	2012-07-24 17:33:56.276859042 +1000
@@ -0,0 +1,2 @@
+# encoding: utf-8
+APP_DIR = File.expand_path('../../..',__FILE__)
--- /dev/null	2012-07-24 17:25:00.561565760 +1000
+++ test/bootstrap/unit.rb.tt	2012-07-24 17:33:56.276859042 +1000
@@ -0,0 +1,3 @@
+# encoding: utf-8
+require_relative 'all'
+
